README.md

```diff
@@ -20,7 +20,7 @@ The support policy is to support the last 2 major versions of Erlang and the thr
 
    ```elixir
    def deps do
-     [{:plug_logger_json, "~> 0.6.0"}]
+     [{:plug_logger_json, "~> 0.7.0"}]
```

Comment:
Can you remove this from the PR? We'll update this when we are ready to publish a new version.